elasticsearch/config/entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -m
+
+# Add elasticsearch as command if needed
+if [ "${1:0:1}" = '-' ]; then
+	set -- elasticsearch "$@"
+fi
+
+if [ "$NODE_NAME" = "" ]; then
+	export NODE_NAME=$HOSTNAME
+fi
+
+if [ "x${ELASTICSEARCH_URL}" = "x" ]; then
+  el_url="https://elasticsearch:9200"
+else
+  el_url="${ELASTICSEARCH_URL}"
+fi
+
+# Run as user "elasticsearch" if the command is "elasticsearch"
```

Comment:
Explain.

Reply:
In this section is unnecessary, this entry belongs to the script that updates the settings.